### PR TITLE
Make it possible to changeable default_max_depth in Raven_Serializer

### PR DESCRIPTION
**Pull request by howyi**
I'd like to refer to the complicated nested JSON request in Sentry.

### CODE REVIEW

**Comment by stayallive** [comment]
Hey @howyi, if you're still interested this also needs some tests to verify it's working correctly.

**Comment by howyi** [comment]
@stayallive Hi.
I'm still interested in this!
OK, I will write a tests, please wait a few days ...

**Comment by howyi** [comment]
@stayallive I wrote some tests. 
Please review this!

File: lib/Raven/Serializer.php [hunk 1]
```diff
@@ -175,4 +185,22 @@ public function setMessageLimit($message_limit)
     {
         $this->message_limit = (int)$message_limit;
     }
+
+    /**
+     * @return int
+     * @codeCoverageIgnore
```
Comment (stayallive) [inline lib/Raven/Serializer.php]: Why whould you ignore the code coverage? You have tests now... 👍

Comment (stayallive) [inline lib/Raven/Serializer.php]: Why whould you ignore the code coverage? You have tests now... 👍

File: lib/Raven/Serializer.php [hunk 2]
```diff
@@ -175,4 +185,22 @@ public function setMessageLimit($message_limit)
...
+     */
+    public function getDefaultMaxDepth()
+    {
+        return $this->default_max_depth;
+    }
+
+    /**
+     * @param int $max_depth
+     * @codeCoverageIgnore
```
Comment (stayallive) [inline lib/Raven/Serializer.php]: Same here, no need to ignore the coverage

**Comment by HazAT** [comment]
Heads up, I change the base to `1.x` since this is the old `master`.
Current `master` branch is the `2.0` version of the SDK.
Version `1.x` will receive minor and patch releases from this branch from now on.

**Comment by howyi** [comment]
@HazAT Thank you!
I am waiting for the review of @stayallive ... 🙄

**Pull request merged**